x-pack/plugins/observability_solution/entity_manager/docs/entity_definitions.md

```diff
@@ -184,6 +184,14 @@ __services_from_logs summary entity__
 The index templates and ingest pipelines created for a given definition are managed by the Entity manager and should not be directly updated. These components still offer extension points that allow customization of each transform through optional components labelled `<component-name>@platform` and `<component-name>@custom` where the former is targeted towards Elastic solution teams and the latter for end users. `@custom` will take precedence when both are defined.
 The extension points allow defining a global component (ie applied to both transforms) with `<definition-id>@custom`, or transform specific components with `<definition-id>-(latest|history)@custom`.
 
-#### Builtin Definitions
+#### Entity discovery
```

Comment:
My understanding is that "entity discovery" is the whole process that a transform does based on an entity definition.

So this is more about "Managed" entity definitions, rather than the discovery part (since the discovery part works the same for custom definitions). 

Can we reword this?